src/test/gtest_main.cpp

```diff
@@ -1,19 +1,58 @@
-#include <sys/stat.h>
-#include <sys/types.h>
+#if __has_include(<filesystem>)
+#include <filesystem>
+#else
+#include <experimental/filesystem>
+#endif
 
 #include <gtest/gtest.h>
+#include "base_test.hpp"
 
 #include "utils/assert.hpp"
 #include "utils/performance_warning.hpp"
 
+#if __has_include(<filesystem>)
+namespace filesystem = std::filesystem;
+#else
+namespace filesystem = std::experimental::filesystem;
+#endif
+
+void create_test_data_directory(std::optional<std::string>& prefix) {
+  if (prefix)
+    opossum::TEST_DATA_PATH = "./" + *prefix + "/test_data/";
+  else
+    opossum::TEST_DATA_PATH = "./test_data/";
```

Comment:
Multiline ifs without braces are frowned upon

Reply:
Also, it is weird for a `create_x_directory` method to (a) delete an existing folder and (b) also set a member of a different class.